### PR TITLE
chore: Add test cases for decorators

### PR DESCRIPTION
**Pull request by bhaskarmelkani**
Add test cases for following.
* Swagger parser
* Swagger Store
* Decorators

### CODE REVIEW

**Comment by bhaskarmelkani** [comment]
👍

File: lib/nodejs/__tests__/swagger-store.test.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+  it('should set and get route', () => {
+    swaggerStore.setRoute('ROUTE', 'DATA');
+    const routeData = swaggerStore.getRoutes('ROUTE');
+    expect(routeData).toBe('DATA');
+  });
+
+  describe('when no route is passed in setRoute', () => {
+    it('should no do anything', () => {
+      swaggerStore.setRoute();
```
Comment (rbarilani) [inline lib/nodejs/__tests__/swagger-store.test.js]: @bhaskarmelkani How do you check that "is not doing anything"?

File: lib/nodejs/swagger/__tests__/swagger.test.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+
+describe('swagger.Swagger', () => {
+
+  // const mockValidate = () => jest.fn()
```
Comment (rbarilani) [inline lib/nodejs/swagger/__tests__/swagger.test.js]: @bhaskarmelkani remove commented code

File: lib/nodejs/swagger/__tests__/swagger.test.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  }));
+
+  const Swagger = require('../swagger');
+
+  it('should expose swagger object', () => {
+    expect(Swagger).toBeDefined();
+  });
+
+  describe('when swagger path is not passed', () => {
+    it('should throw error',async () => {
```
Comment (rbarilani) [inline lib/nodejs/swagger/__tests__/swagger.test.js]: @bhaskarmelkani Why this operation is async?

File: lib/nodejs/__tests__/swagger-store.test.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+
+/**
+ * Mocking crypto.createHash('md5').update('...').digest('hex')
+ */
+const mockDigest = jest.fn(() => {
```
[the remark this answers is not in the document]
Comment (rbarilani) [reply lib/nodejs/__tests__/swagger-store.test.js]: What about

```js
const mockCrypto = {
  createHash: jest.fn(() => {
    return mockCrypto;
  }),
  update: jest.fn(() => {
    return mockCrypto;
  }),
  digest: jest.fn(() => {
    return 'DIGEST'
  })
};
```

[the remark this answers is not in the document]
Comment (rbarilani) [reply lib/nodejs/__tests__/swagger-store.test.js]: then you can ```expect(mockCrypto.update).toHaveBeenCalled()```

File: lib/nodejs/swagger/__tests__/swagger.test.js (hunk 2 above)
Comment (bhaskarmelkani) [reply lib/nodejs/swagger/__tests__/swagger.test.js]: 👍

File: lib/nodejs/swagger/__tests__/swagger.test.js (hunk 3 above)
Comment (bhaskarmelkani) [reply lib/nodejs/swagger/__tests__/swagger.test.js]: Ah yeah, its not required. First was also testing validate with it so was required.
But then seperatad the tests.
Removed

File: lib/nodejs/__tests__/swagger-store.test.js (hunk 1 above)
Comment (bhaskarmelkani) [reply lib/nodejs/__tests__/swagger-store.test.js]: Fixed

File: lib/nodejs/__tests__/swagger-store.test.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (bhaskarmelkani) [reply lib/nodejs/__tests__/swagger-store.test.js]: Looks good to me. 👍

**Comment by bhaskarmelkani** [comment]
👍

**Comment by rbarilani** [comment]
👍

**Comment by bhaskarmelkani** [comment]
👍

**Pull request merged**